### PR TITLE
More minor style fixes

### PR DESCRIPTION
**Pull request by RequiemOfSpirit**
### Changes
- Added external link icons to standalone external links
- Added external link icon to table column headers where all entries in the column are external links
- Updated styles for empty season sub-page placeholder message
- Made minor updates to Brackets page style to make some text stand out less
  - Changed Scheduled Race text to be a very slightly duller white
  - Made time the same slightly duller white color and reduced text size
  - Made "forfeit" in specific a more greyish white color
- Added a transition to show link underlines on hover to remove "underline clutter" on pages with a lot of links

### CODE REVIEW

File: http/templates/bracket_detail.html.tera [hunk 1]
```diff
@@ -25,35 +25,43 @@
...
             <tbody>
                 {% for race in round.races %}
                 <tr title="Race id #{{ race.race_id }}">
                     <td
-                        class="{% if race.player_1.winner %} winner-cell {% elif race.player_1.loser %} loser-cell {% endif %}">
-                        {{ macros::player_detail(name=race.player_1.name) }} ({{ race.player_1.status }})
+                        class="{% if race.player_1.winner %} winner-cell {% elif race.player_1.loser %} loser-cell {% endif %}"
+                    >
+                        {{ macros::player_detail(name=race.player_1.name) }}
+                        <span class="race-time {% if race.player_1.status == 'Forfeit' -%} subdued-text {%- endif %}">
```
[the remark this answers is not in the document]
Comment (RequiemOfSpirit) [reply http/templates/bracket_detail.html.tera:39]: Yeah it doesn't and there's no need to. I just hated seeing a double space when looking at this in the browser console.

File: http/templates/bracket_detail.html.tera [hunk 2]
```diff
@@ -25,35 +25,43 @@
                         Race Status
                     </td>
                     <td>
-                        Restream Link
+                        Restream Link<img class="icon external-link-icon" src="/static/images/external-link.png">
```
[the remark this answers is not in the document]
Comment (RequiemOfSpirit) [reply http/templates/bracket_detail.html.tera:28]: Oh that's a good point. I was originally messing with inline svgs and custom styles on them and then just left them online when I switched to pngs and image tags.

I can probably try making that change everywhere this is used.

File: http/templates/season_qualifiers.html.tera [hunk 3]
```diff
@@ -33,8 +33,8 @@
 {% endblock %}
 
 {% block season_body %}
-<div id="qualifiers" data-season-id="{{ season.id }}" class="seasons-sub-page-container">
-    <div id="qualifiers-wrapper" class="hidden">
+<div id="qualifiers" data-season-id="{{ season.id }}">
```
[the remark this answers is not in the document]
Comment (RequiemOfSpirit) [reply http/templates/season_qualifiers.html.tera:36]: I renamed that class to `stats-container` instead. The sub-page has different margin requirements when there's tables in it compared to when it's empty with just the placeholder.

File: http/templates/bracket_detail.html.tera (hunk 1 above)
[the remark this answers is not in the document]
Comment (FoxLisk) [reply http/templates/bracket_detail.html.tera:39]: lmao fair

**Pull request merged**